x-pack/plugins/aiops/public/kibana_context.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CoreStart } from '@kbn/core/public';
+import { KibanaReactContextValue, useKibana } from '@kbn/kibana-react-plugin/public';
+import type { IStorageWrapper } from '@kbn/kibana-utils-plugin/public';
+import type { AiOpsStartDependencies } from './plugin';
+
+export type StartServices = CoreStart &
+  AiOpsStartDependencies & {
+    storage: IStorageWrapper;
+  };
+export type DataVisualizerKibanaReactContextValue = KibanaReactContextValue<StartServices>;
```

Comment:
Reference to `DataVisualizer` should be changed to `AiOps`.

Reply:
Updated in fd3cbb696ab1d3bc4f6c9cbcfd9f29124e986bf1